frontend/src/Components/WorkstationView.tsx

```diff
@@ -123,51 +143,30 @@ const StatsTable = ({ hostname }: { hostname: string }) => {
           </MenuOptionGroup>
         </MenuList>
       </Menu>
-
-      {validationElim(allStats, {
-        success: (s) => {
-          const gpus = s.flatMap((g) =>
-            g.workstations
-              .filter((w) => w.name === hostname)
-              .flatMap((w) => w.gpus),
-          );
-
-          const tabGpus = transpose(
-            gpus.map((g) => tablify(shownColumns, g)),
-          ).map((r) => all(r));
-
-          return (
-            <TableContainer>
-              <Table variant="striped">
-                <Thead>
-                  <Tr>
-                    <Th>Field</Th>
-                    <Th>GPU 0</Th>
-                  </Tr>
-                </Thead>
-                <Tbody>
-                  {tabGpus.map((fs, i) =>
-                    fs === null ? (
-                      <></>
-                    ) : (
-                      <Tr key={i}>
-                        <Td>{Object.keys(GPU_FIELDS)[i]}</Td>
-                        {fs.map((f, j) => (
-                          <Td key={j}>
-                            {cropString(f, Math.round(35 / fs.length))}
-                          </Td>
-                        ))}
-                      </Tr>
-                    ),
-                  )}
-                </Tbody>
-              </Table>
-            </TableContainer>
-          );
-        },
-        failure: () => <Text>Failure fetching data! Retrying...</Text>,
-        loading: () => <Text>Fetching data...</Text>,
-      })}
+      <TableContainer>
+        <Table variant="striped">
+          <Thead>
+            <Tr>
+              <Th>Field</Th>
+              <Th>GPU 0</Th>
```

Comment:
Oh shit. Not directly relevant to this PR but I forgot I never fixed the headings here (should be as many as there are GPUs connected to that workstation - I think the actual rows should be fine?)

Reply:
Opened an issue https://github.com/gpuctl/gpuctl/issues/251